content/api/node.md

```diff
@@ -1,7 +1,301 @@
+
+``` js-with-links
+const webpack = require("webpack");
+
+const compiler = webpack({
+  // [Configuration Object](/configuration/)
+});
+
+compiler.run((err, [stats](#stats-object)) => {
+  // ...
+});
+```
+
+## Watching
+
+Calling the `watch` method, triggers the webpack runner, but then watches for changes (much like CLI: `webpack --watch`), as soon as webpack detects a change, runs again. Returns an instance of `Watching`.
+
+``` js-with-links
+watch(watchOptions, callback)
+```
+
+``` js-with-links-with-details
+const webpack = require("webpack");
+
+const compiler = webpack({
+  // [Configuration Object](/configuration/)
+});
+
+const watching = compiler.watch({
+  <details><summary>/* [watchOptions](/configuration/watch/#watchoptions) */</summary>
+  aggregateTimeout: 300,
+  poll: undefined
+  </details>
+}, (err, [stats](#stats-object)) => {
+  // Print watch/build result here...
+  console.log(stats);
+});
+```
+
+`Watching` options are [covered in detail here](/configuration/watch/#watchoptions).
+
+### Close `Watching`
+
+The `watch` method returns a `Watching` instance that exposes `.close(callback)` method. Calling this method will end watching:
+
+``` js
+watching.close(() => {
+  console.log("Watching Ended.");
+});
+```
+
+T> It’s not allowed to watch or run again before the existing watcher has been closed or invalidated.
+
+### Invalidate `Watching`
+
+Manually invalidate the current compiling round, but don’t stop watching.
+
+``` js
+watching.invalidate(() => {
+  console.warn("Invalidated.");
+});
+```
+
+## Stats Object
+
+The `stats` object that is passed as a second argument of the [`webpack()`](#webpack-) callback, is a good source of information about the code compilation process. It includes:
+
+- Errors and Warnings (if any)
+- Timings
+- Module and Chunk information
+- etc.
+
+The [webpack CLI](/api/cli) uses this information to display a nicely formatted output in your console.
+
+This object exposes these methods:
+
+### `stats.hasErrors()`
+
+Can be used to check if there were errors while compiling. Returns `true` or `false`.
+
+### `stats.hasWarnings()`
+
+Can be used to check if there were warnings while compiling. Returns `true` or `false`.
+
+### `stats.toJSON(options)`
+
+Returns compilation information as a JSON object. `options` can be either a string to specify the granularity level or an object of options in which you can customize the output completely. E.g:
+
+``` js
+stats.toJSON("minimal");
+```
+
+| Option | Alternative | Description |
+|--------|-------------|-------------|
+| `none`        | `false` | Output nothing |
+| `errors-only` | *none*  | Only output when errors happen |
+| `minimal`     | *none*  | Only output when errors or new compilation happen |
+| `normal`      | `true`  | Standard output |
+| `verbose`     | *none*  | Output everything |
+
+Or an object of options. **Note** that all items are optional:
+
+``` js
+stats.toJSON({
```

Comment:
Same for this, these options are also documented in the stats page. You have documented them better though (with a description). Could you move this to the stats page? I'd only show a quick demo of one or two options here.